test/ctia/test_helpers/generators/schemas.clj

```diff
@@ -0,0 +1,64 @@
+(ns ctia.test-helpers.generators.schemas
+  (:require [clojure.test.check.generators :as gen]
+            [ctia.schemas
+             [feedback :refer [Feedback]]
+             [identity :refer [Identity]]
+             [verdict :refer [Verdict]]]
+            [ctia.test-helpers.generators.common :refer [generate-entity]]
+            [ctia.test-helpers.generators.schemas
+             [actor-generators :as ag]
+             [campaign-generators :as cg]
+             [coa-generators :as og]
+             [exploit-target-generators :as eg]
+             [incident-generators :as ig]
+             [indicator-generators :as ng]
+             [judgement-generators :as jg]
+             [sighting-generators :as sg]
+             [ttp-generators :as tg]]))
+
+(def gen-new-indicator-with-new-sightings
+  (gen/fmap
+   (fn [[indicator sightings]]
+     [indicator
+      (map (fn [sighting]
+             (assoc sighting :indicators
+                    [{:indicator_id (:id indicator)}]))
```

Comment:
This is wrong.  The :indicator_id values need to be URLs ('long IDs'), and the store methods need to handle them correctly.